### PR TITLE
refactor: PrayCard History 수정

### PR DESCRIPTION
**Pull request by myeongjunkim**
image

# 작업 설명
현재 UI 는 깨지는 부분도 많고 기존 UI 들과 다른 느낌이라 기존 UI 느낌으로 대체합니다.

# 체크리스트

- [x]  기도카드 리스트 UI 및 반응형 수정

### CODE REVIEW

**Comment by myeongjunkim** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/296?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#296** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/296?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#295** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/295?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @myeongjunkim and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/profile/SettingDialog.tsx [hunk 1]
```diff
@@ -56,44 +53,18 @@ const SettingDialog = () => {
...
-        <div className="w-full flex flex-col gap-4">
-          <Skeleton className="w-full h-[55px] flex items-center gap-4 p-4 bg-gray-300 rounded-xl" />
-          <Skeleton className="w-full h-[55px] flex items-center gap-4 p-4 bg-gray-300 rounded-xl" />
-          <Skeleton className="w-full h-[55px] flex items-center gap-4 p-4 bg-gray-300 rounded-xl" />
-          <Skeleton className="w-full h-[55px] flex items-center gap-4 p-4 bg-gray-300 rounded-xl" />
-        </div>
-      </div>
-    );
-  }
-
```
Comment (myeongjunkim) [inline src/components/profile/SettingDialog.tsx:96]: dialog 에서는 로딩처리가 필요 없을 것 같아 제거합니다.

File: src/components/profile/PrayCardHistoryPrayList.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const PrayCardHistoryPrayList: React.FC = () => {
             />
           </div>
           <div className="flex flex-col text-gray-400 text-sm text-center">
-            <p>아직 기도해 준 친구가 없어요 😭</p>
+            <p>기도해 준 친구가 없어요 😭</p>
```
Comment (yaminsu5674) [inline src/components/profile/PrayCardHistoryPrayList.tsx:30]: ㅋㅋㅋ 예리하시네요 bb

**Review by yaminsu5674 — APPROVED** [review]
굿굿입니다!

**Review by treephesians — APPROVED** [review]
훨씬 깔끔해졌네요 감사합니다!!

**Pull request merged**